### PR TITLE
Updated with new pattern

### PR DESCRIPTION
**Pull request by russellseymour**
Updated NGinx with new AzDo pattern:

 - Added InSpec controls
 - Removed BATS tests
 - Added `run` hook which is needed for testing

### CODE REVIEW

File: controls/nginx_service.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+plan_name = input('plan_name', value: 'nginx')
+plan_ident = "#{ENV['HAB_ORIGIN']}/#{plan_name}"
+
+control 'base-plans-nginx-service' do
+    impact 1.0
+    title 'Ensure Nginx is running as a service'
+    desc ''
+
+    # Get the ident of the package
+    status_cmd = command("hab svc status #{plan_ident}")
```
Comment (skpaterson) [inline controls/nginx_service.rb:13]: Might be worth ensuring the status call is successful here e.g. 
```
    describe status_cmd do
        its('exit_status') { should eq 0 }
        its('stdout') { should_not be_empty }
    end
```

**Review by skpaterson — APPROVED** [review]
Great work, thanks @russellseymour!  One small comment but happy to proceed as is.

File: controls/nginx_service.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    svc_status = status_cmd.stdout.strip.split(/\s+/)
+
+    # Test that the service is running
+    # Ideally this should test the type, desired and state - however
```
Comment (gavindidrichsen) [inline controls/nginx_service.rb:20]: Entirely up to you @russellseymour, but if you look in the memcached or minio plans you will find a regex that will check whether the service is up:

``its('stdout') { should match /(?<package>#{plan_pkg_ident})\s+(?<type>standalone)\s+(?<desired>up)\s+(?<state>up)/ }``

File: controls/nginx_service.rb (hunk 1 above)
Comment (russellseymour) [reply controls/nginx_service.rb:13]: Have added this :+1:

File: controls/nginx_service.rb (hunk 2 above)
Comment (russellseymour) [reply controls/nginx_service.rb:20]: Thanks have added this :+1:

**Pull request merged**